File: p2/Estrutura de Dados/4 - Pilha Sequencial/Aluno.py
class Aluno:
    def __init__(self,matricula,nome):
        self.matricula = matricula
        self.nome = nome
    def __str__(self):
        return f'{self.nome}, matricula {self.matricula}'

    def __eq__(self, other):
        return self.matricula == other.matricula

File: p3/POO/2 - Códigos em Sala/aluno/Aluno.java
package aluno;

public class Aluno {
    // Criação dos atributos
    private String nome;
    private Integer nota1;
    private Integer nota2;

    public Aluno(String texto, int nota1, int nota2) {
        this.nome = texto;
        this.nota1 = nota1;
        this.nota2 = nota2;
    }    
    
    // Metodos get
    public String getNome() {
        return nome;
    }

    public Integer getMedia() {
        return (nota1 + nota2) / 2;
    }

    public String getSituacao() {
        if (this.getMedia() >= 70)
            return "Aprovado";
        else if (this.getMedia() >= 40)
            return "Reprovado";
        else return "Final";
    }
}
